packages/node-sdk/example/app.ts

```diff
@@ -0,0 +1,59 @@
+import express from "express";
+import { bucket } from "./bucket";
+
+export const app = express();
+
+app.use(express.json());
+app.use((req, res, next) => {
+  const bucketUser = bucket
+    .withUser(req.headers["x-bucket-user-id"] as string, {
+      attributes: {
+        role: req.headers["x-bucket-is-admin"] ? "admin" : "user",
+      },
+    })
+    .withCompany(req.headers["x-bucket-company-id"] as string, {
+      attributes: {
+        betaUser: !!req.headers["x-bucket-company-beta-user"],
+      },
+    });
+
+  res.locals.bucketUser = bucketUser;
+  next();
+});
+
+const todos = ["Buy milk", "Walk the dog"];
+
+app.get("/", (req, res) => {
+  res.locals.bucketUser.track("Front Page Viewed");
+  res.send("Hello World");
+});
+
+app.get("/todos", (req, res) => {
+  const showTodosFlag = await res.locals.bucketUser.getFlag("show-todos");
```

Comment:
Wait, did I add a separate `getFlag`? I though not. Also, we discussed the `global` flag type, I opted to make the `getFlags` generic. Should we exemplify that by declaring a `type AppFlags = 'show-todos' | 'something'` and

Should we make the `BucketClient` generic or just the flag getting function?